Scripts/Python/kdshGlowInTheDark.py

```diff
@@ -284,7 +284,7 @@ def OnNotify(self,state,id,events):
 
             elif id == rgnEnterSubBtm.id:
                 PtDebugPrint("You stepped on the elevator at the bottom. Joining subworld.")
-                rgnExitSubBtm.disable()
+                #rgnExitSubBtm.disable()
```

Comment:
```suggestion
```
No need to comment out something we want to remove, as version control tracks that for us nicely. :)  It's best to simply remove the line.